src/components/ActorsTodo/ActorsTodo.tsx

```diff
@@ -0,0 +1,69 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+import * as S from "./ActorsTodo.Styled";
+import { useState } from "react";
+
+interface TypesForActors {
+	id: number;
+	name: string;
+}
+
+function ActorsTodo() {
+	const [actorsList, setActorsList] = useState<TypesForActors[]>([]);
+	const [newActor, setNewActor] = useState("");
+
+	const addActor = () => {
+		if (newActor.trim() !== "") {
+			const actorsObject = {
+				id: Date.now(),
+				name: newActor,
+			};
+
+			setActorsList([...actorsList, actorsObject]);
+			setNewActor("");
+
+			console.log(actorsList);
+		}
+	};
+
+	const deleteActor = (id: number) => {
+		setActorsList(actorsList.filter((item) => item.id !== id));
+	};
+
+	const clearAll = () => {
+		setActorsList([]);
+	};
+
+	return (
+		<S.Todo>
+			<S.InputField>
+				<S.InputForActor
+					type="text"
+					value={newActor}
+					onChange={(e) => setNewActor(e.target.value)}
+					placeholder="Dodaj aktora"
+				/>
+				<S.ButtonContainer>
+					<S.AddBtn onClick={addActor}>Dodaj</S.AddBtn>
+					<S.ClearBtn onClick={clearAll}>Usuń</S.ClearBtn>
```

Comment:
Co myślisz o tym, aby zrobić przycisk usuń jako disabled w przypadku gdy nie mamy żadnych aktorów? detal, ale warto pamiętać o takich detalach. Po co umożliwiać userowi czyszczenie, skoro lista jest pusta.

To samo tyczy się dodawania, jeżeli nie wpisał żadnego znaku do inputa, to możemy mu zablokować przycisk Dodaj.